### PR TITLE
Upgrade SDK and dependencies

### PR DESCRIPTION
**Pull request by empovit**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Build the manager binary
-FROM golang:1.18 as builder
+FROM golang:1.19 as builder
```
Comment (mresvanis) [inline Dockerfile:2]: Let's base our builder images on ubi8

```
FROM registry.access.redhat.com/ubi8/go-toolset:1.18 as builder
```

File: Makefile [hunk 2]
```diff
@@ -178,14 +197,14 @@ GOLANGCI_LINT ?= $(LOCALBIN)/golangci-lint
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v3.8.7
```
Comment (mresvanis) [inline Makefile:199]: Let's include also a kustomize version update to [4.5.7](https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv4.5.7).

**Comment by mresvanis** [comment]
We could also update the OPM binary version to [1.26.2](https://github.com/operator-framework/operator-registry/releases/tag/v1.26.2) in the [Makefile](https://github.com/empovit/fdo-operator/blob/main/Makefile#L229).

File: Dockerfile (hunk 1 above)
Comment (empovit) [reply Dockerfile:2]: Hmm, the updated SDK (the OpenShift version) requires Go 1.19, which doesn't seem available with UBI yet.

File: .gitignore [hunk 3]
```diff
@@ -9,6 +9,7 @@ keys/
 *.dylib
 bin
 testbin/*
+Dockerfile.cross
```
Comment (mresvanis) [inline .gitignore:12]: We won't be needing this line (for now) :)

**Pull request merged**